wallet/test/wallet.go

```diff
@@ -34,39 +32,17 @@ type UnlockedAccount func() (wallet.Account, error)
 
 // Setup provides all objects needed for the generic tests.
 type Setup struct {
-	Backend wallet.Backend // backend implementation
-	Wallet  wallet.Wallet  // the wallet instance used for testing
-	Address wallet.Address // an address of an account in the test wallet
-	// Address tests
-	AddressEncoded []byte // a valid nonzero address not in the wallet
-	// Signature tests
-	DataToSign []byte
+	Backend        wallet.Backend // backend implementation
+	Wallet         wallet.Wallet  // the wallet instance used for testing
+	Address        wallet.Address // an address of an account in the test wallet
+	DataToSign     []byte         // some data to sign
+	AddressEncoded []byte         // a valid nonzero address not in the wallet
 }
 
-type NewSetupFunc func(rng *rand.Rand) *Setup
```

Comment:
If i see it correctly, you added this function in cc85cc9fd333cc6645aa5f8340320fb6475748ec.  
So maybe you can fix it up.